src/assets/wise5/components/common/feedbackRule/edit-feedback-rules/edit-feedback-rules.component.ts

```diff
@@ -64,7 +64,7 @@ export class EditFeedbackRulesComponent implements OnInit {
     this.projectService.nodeChanged();
   }
 
-  private createNewFeedbackRule(): any {
+  protected createNewFeedbackRule(): any {
```

Comment:
Will also need to change this class's feedbackRules declaration to this:
```@Input() feedbackRules: Partial<FeedbackRule>[] = [];```

```suggestion
  protected createNewFeedbackRule(): Partial<FeedbackRule> {
```